src/Momento.Sdk/ICacheClient.cs

```diff
@@ -452,6 +452,16 @@ public interface ICacheClient : IDisposable
     /// <inheritdoc cref="DictionaryRemoveFieldsAsync(string, string, IEnumerable{byte[]})"/>
     public Task<CacheDictionaryRemoveFieldsResponse> DictionaryRemoveFieldsAsync(string cacheName, string dictionaryName, IEnumerable<string> fields);
 
+    /// <summary>
+    /// Calculate the length of a dictionary in the cache.
+    ///
+    /// A dictionary that does not exist is interpreted to have length 0.
```

Comment:
A dictionary that does not exist will be a miss. The list length in this repo returns length 0 for a miss, which is non-standard now.